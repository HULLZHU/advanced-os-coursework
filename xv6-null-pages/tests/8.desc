testing whether forking the process keeps the same protection bits.

When you run the test_8.c in xv6, the program will crash & you should be getting an error message like:
    "pid 23 test_8: trap 14 err 7 on cpu 0 eip 0x1047 addr 0x1000--kill proc"