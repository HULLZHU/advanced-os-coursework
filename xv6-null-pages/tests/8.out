XV6_TEST_OUTPUT: Return value of mprotect on memory 0x1000: 0
XV6_TEST_OUTPUT: Inside child process
